components/MintPage/MintDay/Modals/MintMoreModal.tsx

```diff
@@ -10,26 +10,13 @@ import { useMintProvider } from "../../../../providers/MintProvider"
 
 interface MintMoreModalProps extends IMintModal {}
 
-const MintMoreModal: FC<MintMoreModalProps> = ({
-  isModalVisible,
-  toggleIsVisible,
-  loading,
-  coreMintFunc,
-  handleLoading,
-  openSuccessModal,
-}) => {
+const MintMoreModal: FC<MintMoreModalProps> = ({ isModalVisible, toggleIsVisible, loading }) => {
   const isXl = useMediaQuery("(max-width: 1150px)")
 
-  const { checkNetwork, refetchInformation, leftQuantityCount, lockedCntOfCre8or } =
-    useMintProvider()
+  const { leftQuantityCount, lockedCntOfCre8or } = useMintProvider()
```

Comment:
you need locked and unlocked id's correct? I am going to add that logic to initalData retrieval using multicall.